sys/fuchsia/threads.txt

```diff
@@ -6,11 +6,11 @@ include <zircon/syscalls/debug.h>
 
 resource zx_thread[zx_task]
 
-zx_thread_create(process zx_process, name ptr[in, string], name_len len[name], options const[0], out ptr[out, zx_thread])
 zx_thread_exit()
+zx_thread_create(process zx_process, name ptr[in, string], name_size len[name], options const[0], out ptr[out, zx_thread])
+zx_thread_start(handle zx_thread, thread_entry int64, stack int64, arg1 const[0], arg2 const[0])
```

Comment:
Here is a place where the previous definition was a better fit, as it would have allowed syzkaller to create a bunch of code for the new thread to run.